sdk/storage/Microsoft.Azure.Management.Storage/src/Microsoft.Azure.Management.Storage.csproj

```diff
@@ -10,7 +10,7 @@
 			Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Storage" (https://www.nuget.org/packages/Azure.ResourceManager.Storage/). Please see the package changelog for more information.
 		</Description>
 		<AssemblyName>Microsoft.Azure.Management.Storage</AssemblyName>
-		<Version>22.0.0</Version>
+		<Version>22.6.0</Version>
```

Comment:
In the change log, it's said the version is `23.0.0`. Which one is correct?

Reply:
Fixed. Thanks for the catch!